docs/source/tools/navigator/console.rst

```diff
@@ -491,4 +491,4 @@ To run Navigator against a secured Digital Asset Ledger, configure TLS certifica
 
 Details of these parameters are explained in the command line help::
 
-  da run navigator -- --help
+  daml navigator -- --help
```

Comment:
```suggestion
  daml navigator --help
```
We should confirm that `--help` doesn’t get eaten by something but at least in theory the `--` should be unnecessary.

Reply:
Just checked and the `--` in the middle is indeed unnecessary.